File: Data Structures & Algorithms/maximum-subarray/submission-0.java
class Solution {
    public int maxSubArray(int[] nums) {
        int cur = 0;
        int max = Integer.MIN_VALUE;
        for(int i : nums) {
            cur = cur + i;
            max = Math.max(cur,max);
            if(cur < 0)
            cur = 0;
        }
        return max;
    }
}

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        maxCount = 0
        curCount = 0
        n = len(nums)
        for i in range(n):
            if nums[i] == 1:
                curCount += 1
            else:
                maxCount = max(curCount,maxCount);
                curCount = 0
        return max(curCount,maxCount)
        